projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-control-checkbox-option-model.ts

```diff
@@ -0,0 +1,37 @@
+import { TableCheckboxOptions, TableControlOption } from '@hypertrace/components';
+import { BOOLEAN_PROPERTY, Model, ModelProperty } from '@hypertrace/hyperdash';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { TableWidgetControlModelBase } from './table-widget-control-model-base';
+
+@Model({
+  type: 'table-widget-checkbox-option'
+})
+export class TableWidgetControlCheckboxOptionModel extends TableWidgetControlModelBase {
+  @ModelProperty({
+    key: 'checked',
+    displayName: 'Checked',
+    type: BOOLEAN_PROPERTY.type,
+    required: false
+  })
+  public checked: boolean = false;
```

Comment:
so a checkbox has two options, true and false. Is this model just to associate metadata with each one?